app/src/server/trpc/routers/service.ts

```diff
@@ -37,7 +37,11 @@ export const serviceRouter = router({
         resArtifacts => resArtifacts.json() as Promise<fhir4.Bundle<FhirArtifact>>
       );
       const artifactList = artifactBundle.entry?.map(entry => ({
-        label: entry.resource?.name || entry.resource?.id || '',
+        label:
+          entry.resource?.name?.concat(`|${entry.resource?.version}`) ||
+          entry.resource?.name ||
```

Comment:
I don't think we need this `entry.resource?.name ||` line because the first line is already checking whether there's a name field.

Reply:
Good point! I thought we would want to display the name if there wasn't a name AND a version but I don't think that'll ever be the case (we make sure there is a version on upload).